man/systemd.net-naming-scheme.xml

```diff
@@ -457,6 +468,17 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><constant>v254</constant></term>
+
+          <listitem><para>Naming was changed for SR-IOV virtual device representors.</para>
+
+          <para>The <literal>r<replaceable>port</replaceable></literal> suffix was added to differentiate SR-IOV
```

Comment:
https://github.com/systemd/systemd/pull/28118#discussion_r1246865752
Then, please update this line: port -> slot